src/applications/mhv-secure-messaging/components/ComposeForm/RecipientsSelect.jsx

```diff
@@ -60,6 +62,8 @@ const RecipientsSelect = ({
   const isSignatureRequiredRef = useRef();
   isSignatureRequiredRef.current = isSignatureRequired;
 
+  const { isComboBoxEnabled } = useFeatureToggles();
```

Comment:
`useFeatureToggles` exports `featureTogglesLoading`. Do you think it would be useful to apply a check here to avoid re-rendering?